console/console.properties

```diff
@@ -310,9 +310,20 @@ AreasGroup=INSEE_DEP
 # default: false
 #saslEnabled=false
 
+# Activates analytics 
+# if set to true, the console will add all links to analytics and execute XHR requests to analytics
+# default: true
+# analyticsEnabled=true
```

Comment:
if `analyticsEnabled` has an effect on several modules (and it does, to my understanding: header, console), then it should probably belong to defaults.properties at the root of this repository.

Reply:
like if administrators reading config files in the datadir had any idea what the "georchestra gateway" was... </devil's advocate>

Reply:
Soon you will !